ani-cli

```diff
@@ -235,16 +230,16 @@ update_history() {
 
 download() {
     case $1 in
-        *m3u8*) ffmpeg -loglevel error -stats -i "$1" -c copy "$download_dir/$2.mp4" ;;
-        *) if curl -s -m 5 "$1" | grep -q '^#EXTM3U'; then
-            ffmpeg -loglevel error -stats -i "$1" -c copy "$download_dir/$2.mp4"
-        else
-            if uname -a | grep -q "ish"; then
-                curl --output-dir "$download_dir" -o "$2.mp4" "$1"
+        *m3u8*)
+            if command -v "hls"; then
+                hls -n 300 -ro "$download_dir/$2" "$1"
```

Comment:
I wonder if there's a way to do this with `dep_ch` and `player_fn`. Especially that in this state not having hls on ish would lead to an undefined state. (I suppose the check was in place because ffmpeg is unavailable on ish?)

Reply:
check was in place in case someone didn't install the script, and yes you can use dep_ch I was writing the code when body temp was high, so I didn't think about that